CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - fix random crashes on quote reply #4337
 - avoid drafts in readonly chats #4349
 - fullscreen images getting cropped a little #4402, #4385
+- fix chat "scrolls up" right after switching #4404
```

Comment:
normally the number here refers to the pr, not the issue.

But then again, in my personal opinion this "add pr number to pr" rule is annoying.... (but I also see some tiny  use in having that info)